src/components/common/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { UseFormRegisterReturn } from 'react-hook-form';
+import { TextAreaWrapper, TextAreaLabel, TextAreaContainer } from './TextAreaStyle';
+
+interface ITextAreaProps {
+  value?: string;
+  register?: UseFormRegisterReturn;
+  label: string;
+  name: string;
+  onChange?: (e: React.ChangeEvent<HTMLTextAreaElement>) => void;
+}
+
+const TextArea: React.FC<ITextAreaProps> = ({ register, onChange, label, ...props }) => {
+  console.log(register);
+  return (
+    <TextAreaWrapper>
+      <TextAreaLabel>{label}</TextAreaLabel>
+      <TextAreaContainer {...props} {...register} onChange={onChange} />
```

Comment:
why this is called container if it is textarea itself?

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/27/commits/35d1051dd92f9591600c90270a824d4eec0360e7#diff-526d33de93204a6fc68e39d8ecbdf82532a7206dcd9c3c1648b3a00a5d6b35f8